CovidWatch/build.gradle

```diff
@@ -14,14 +14,15 @@ buildscript {
 
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
+        //implementation 'org.covidwatch.libcontacttracing:covidwatch-android:0.1.0'
```

Comment:
I guess we don't need this for now and the jitpack but like the idea of having a separate repository for the library in the future.